xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -246,28 +252,36 @@ void CXBMCApp::onResume()
     m_applications.clear();
   }
 
+  if (m_bResumePlayback && g_application.GetAppPlayer().IsPlaying())
+  {
+    if (g_application.GetAppPlayer().HasVideo())
```

Comment:
Is this just with video, not music?

Reply:
These are the same conditions we pause playback (in onPause) - they haven't changed.  Music should continue playing, even if kodi is invisible.